iceoryx_utils/test/moduletests/test_message_queue.cpp

```diff
@@ -24,7 +24,11 @@ using namespace ::testing;
 using namespace iox;
 using namespace iox::posix;
 
+#if defined(__APPLE__)
+using IpcChannel = UnixDomainSocket;
+#else
 using IpcChannel = MessageQueue;
```

Comment:
why not just IpcChannel = IpcChannelType?

Reply:
done